pkg/monitor/cluster/summary.go

```diff
@@ -41,11 +41,12 @@ func (mon *Monitor) emitSummary(ctx context.Context) error {
 	}
 
 	mon.emitGauge("cluster.summary", 1, map[string]string{
-		"actualVersion":     actualVersion(cv),
-		"desiredVersion":    desiredVersion(cv),
-		"masterCount":       strconv.Itoa(masterCount),
-		"workerCount":       strconv.Itoa(workerCount),
-		"provisioningState": mon.oc.Properties.ProvisioningState.String(),
+		"actualVersion":           actualVersion(cv),
+		"desiredVersion":          desiredVersion(cv),
+		"masterCount":             strconv.Itoa(masterCount),
+		"workerCount":             strconv.Itoa(workerCount),
+		"provisioningState":       mon.oc.Properties.ProvisioningState.String(),
+		"failedProvisioningState": mon.oc.Properties.FailedProvisioningState.String(),
```

Comment:
I was initially trying to use `backend.openshiftcluster.count`, but numbers looked inadequate. I think it is because we emit the backend metric on each `EndLease` call with terminal state: so five delete attempts on a same cluster in failed/deleting state would result in 5 "clusters" on the dashbaord.